python/src/IntegrationExpansion_doc.i.in

```diff
@@ -0,0 +1,57 @@
+%feature("docstring") OT::IntegrationExpansion
+"L2 approximation on an orthonormal basis using least-squares and a fixed basis.
+
+Available constructors:
+    IntegrationExpansion(*inputSample, outputSample, distribution*)
+
+    IntegrationExpansion(*inputSample, weights, outputSample, distribution*)
+
+    IntegrationExpansion(*inputSample, outputSample, distribution, basis, basisSize*)
+
+    IntegrationExpansion(*inputSample, weights, outputSample, distribution, basis, basisSize*)
+
+Parameters
+----------
+inputSample : 2-d sequence of float of size :math:`n>0`
+    Input sample.
+weight : sequence of float of size :math:`n>0`, optional
+    Input weights. If not given, the default value is a uniform weight equals to :math:`1/n`.
+outputSample : 2-d sequence of floats of size :math:`n>0`
+    Output sample.
+distribution : :class:`~openturns.Distribution`
+    The distribution associated to the weighted sample *(inputSample, weights)*
+basis : :class:`~openturns.OrthogonalBasis`, optional
+    The orthonormal basis of the vector space on which the approximation is built.
+    If not given, a basis is built using an hyperbolic enumerate function
+    :class:`~openturns.HyperbolicAnisotropicEnumerateFunction` associated to the
+    :math:`q-`norm specified by the 'FunctionalChaosAlgorithm-QNorm' key in
```

Comment:
This is wrongly formatted, but I do not see exactly why:

![image](https://user-images.githubusercontent.com/31351465/235109246-fe062316-1b35-4f77-b9b6-3c450f0acf3b.png)